usaspending_api/database_scripts/matview_generator/mv_loan_award_search.json

```diff
@@ -110,6 +111,16 @@
     "    ON (awards.latest_transaction_id = transaction_fabs.transaction_id AND latest_transaction.is_fpds = false)",
     "LEFT OUTER JOIN",
     "  (SELECT",
+    "    award_id, jsonb_agg(DISTINCT jsonb_build_object('cfda_number', cfda_number, 'cfda_program_title', cfda_title))::text AS cfdas",
```

Comment:
Do you see any need to filter on the `cfda_program_title`?

You are capturing both the `cfda_number` and `cfda_program_title` here but since the `cfda_program_title` can have spaces that is triggering multiple tokens by the `stop analyzer`. For example, querying
```
{
  "query": {
    "match": {
      "cfdas.breakdown": "MEDICAL this is gibberish"
    }
  }
}
```
will return results matching on `MEDICAL` such as all Awards with `cfda_number = '93.778'` which has a `cfda_title = 'MEDICAL ASSISTANCE PROGRAM'`.

This works fine for the `cfda_number` since it registers "93.778" as a single token.